getting-started/contribute.md

```diff
@@ -26,7 +26,7 @@ If you have an IT or CS background and can code, go to [#monero-dev](irc://chat.
 Secure the network and help keep Monero decentralized, it's simple to do, and only requires some bandwidth. Learn [how to set up a monero node here.](/getting-started/running) You can also run a node on a VPS, [guide on how to do that is here.](/knowledge-base/user-guides/vps_run_node)
 
 ### Mine 
-It is still possible to mine, even on a cpu. If you have some spare hardware, you are actively adding to it's decentralisation. Find more information on mining Monero in [this thread on Bitcointalk.](https://bitcointalk.org/index.php?topic=653467)
+It is still possible to mine, even on a cpu. If you have some spare hardware, you are actively adding to it's decentralisation. Find more information on mining Monero in [here.](https://forum.getmonero.org/13/mining-software-and-pools)
```

Comment:
The Bitcointalk thread is still active, I'd suggest keeping it and just appending the forum link like so:

> It is still possible to mine, even on a cpu. If you have some spare hardware, you are actively adding to it's decentralisation. Find more information on mining Monero [on Bitcointalk](https://bitcointalk.org/index.php?topic=653467) or [on the Monero forum](https://forum.getmonero.org/13/mining-software-and-pools).